<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextToEncrypt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
	color: rgb(0, 0, 0);
	border: 2px solid rgb(21, 236, 255);
    border-radius: 25px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(255, 255, 255);
	font: 11pt &quot;Montserrat&quot;;
	

}


</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEncrypted">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
	border: 2px solid rgb(0, 0, 102);
    border-radius: 25px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(255, 255, 255);
	font: 11pt &quot;Montserrat&quot;;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditShift">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(0, 0, 99);
    border-radius: 15px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(255, 255, 255);
	
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid rgb(170, 0, 255);
    border-radius: 10px;
	background-color: rgb(255, 255, 255);
	font: 500 9pt &quot;Montserrat Medium&quot;;
	color: rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QComboBox" name="encryptiondecryptionComboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 2px solid rgb(170, 0, 255);
    border-radius: 15px;
	padding-left: 10px;
	padding-right: 20px;
	background: white;
	font: 500 9pt &quot;Montserrat Medium&quot;;
	color: rgb(0, 0, 0);
	
	
}

QComboBox::down-arrow {
    image: url(:/down-arrow.png);
    width: 12px;
    height: 12px;
	
	
}

QComboBox::drop-down {
    border: none;
	box-shadow: none;
	padding-right: 10px;
}

QComboBox QAbstractItemView {
    padding: 2px; 
    margin: 2px;
	padding-left: 5px;
	padding-right: 5px;
	border: none;
	box-shadow: none;
}

QComboBox QAbstractItemView::item {
	margin: 2px;
	text-align: center;
	height: 30 px;
	border: 2px solid rgb(170, 0, 255);
	border-radius: 15px;
	box-shadow: none;
	color: rgb(0, 0, 0);

}

QComboBox QAbstractItemView::item {
    background-color: white;

}

QComboBox QAbstractItemView::item:hover {
    background-color: rgb(170, 0, 255);
	outline: none;

color: white;
}

QComboBox QAbstractItemView::item:selected {
    background-color: rgb(170, 0, 255);
	outline: none;
border: none;

selection-background-color: transparent;
color: white;
}

</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Encryption</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decryption</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File encryption</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File decryption</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 900 48pt &quot;Montserrat Black&quot;;
	color: rgb(0, 0, 99);
	text-shadow: 2px 2px 2px rgb(0, 0, 127);
}


</string>
    </property>
    <property name="text">
     <string>Caesar Cipher</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="decryptCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="visible">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	font: 500 9pt &quot;Montserrat Medium&quot;;
	color: rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Shift unknown</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
